docs/walkthrough.md

```diff
@@ -138,7 +138,16 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address of the `Service` (and just the IP - no ports or `http://` prefixes)
+that sits in front of the API server pod(s). If you installed the catalog with one of the 
+`helm install` commands above, then this service will be called `catalog-catalog-apiserver`, 
```

Comment:
@duglin unfortunately, on minikube, `kubectl endpoints` doesn't work and users will have to use the VM IP that's returned by `minikube service`. The IP is highly dependent on the deployment.

In this PR, I can mention that the way you get this IP is dependent on your deployment, but I'd like to leave deployment-specific instructions for a later PR